src/main/cheat_code.cpp

```diff
@@ -1795,7 +1889,7 @@ static const CheatCodeDefault_t s_cheatsListLevelDefault[] =
     {"2player", twoPlayer, true},
 
     {"wariotime", warioTime, true},
-    {"carkeys", carKeys, true},
+    {"wherearemycarkeys", carKeys, true},
```

Comment:
This was a shortener made by @0lhi, but if you add another cheat that conflicts with this, that's fine.